src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar_js.html.twig

```diff
@@ -1,117 +1,17 @@
-<div id="sfwdt{{ token }}" class="sf-toolbar sf-display-none"></div>
+<div id="sfwdt-container" class="sf-toolbar sf-display-none"></div>
```

Comment:
not good. The toolbar was always meant to avoid static ids (@fabpot I think there was the possibility to inject multiple toolbars in the page thanks to this. Is it still needed ?)

Reply:
might be preserved and select toolbar by class name :+1:

Reply:
@ro0NL The `Sfjs.load()` expects an element ID as the first parameter because it uses `document.getElementById()` to get the container.

Another other solution would be to let the div's id include the `token` and then remember the very first token that was passed to this function with a static variable. This would work (AFAIK) because the first call would always be the call from the initial static page load and all following calls would be from ajax requests. This seems like an ugly and potentially buggy hack to say the least. Hopefully the static ID isn't still needed?

Reply:
another option: make `loadToolbar` take 2 arguments: the token, and the container id.
The initial loading would pass them both based on the Twig variable. And the ajax replacement would read the container id from the DOM (looking at the ancestor of the ajax panel)